packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -120,144 +120,144 @@ const ClinicalResultBloodPressure = (props) => {
+    originalDiastolic: diastolic && diastolic.conceptDisplay,
+  };
+  const compareDatetimeDisplays = {
+    originalSystolic: systolic && systolic.datetimeDisplay,
+    originalDiastolic: diastolic && diastolic.datetimeDisplay,
+  };
 
-    let iconGroupDisplayElement = null;
+  let iconGroupDisplayElement = null;
 
-    const noDataSystolic = (systolic && systolic.resultNoData === true);
-    if (systolic && !noDataSystolic) {
-      const {
-        result,
-        status,
-        conceptDisplay,
-        datetimeDisplay,
-      } = systolic;
+  const noDataSystolic = (systolic && systolic.resultNoData === true);
+  if (systolic && !noDataSystolic) {
+    const {
+      result,
+      status,
+      conceptDisplay,
+      datetimeDisplay,
+    } = systolic;
 
-      if (!isEmpty(result.unit)) { compareUnits.systolic = result.unit.trim().toLowerCase(); }
-      if (!isEmpty(status)) { compareStatusIsInError.systolic = checkIsStatusInError(status); }
-      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.systolic = conceptDisplay.trim().toLowerCase(); }
-      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.systolic = datetimeDisplay.trim().toLowerCase(); }
-    }
+    if (!isEmpty(result.unit)) { compareUnits.systolic = result.unit.trim().toLowerCase(); }
+    if (!isEmpty(status)) { compareStatusIsInError.systolic = checkIsStatusInError(status); }
+    if (!isEmpty(conceptDisplay)) { compareConceptDisplays.systolic = conceptDisplay.trim().toLowerCase(); }
+    if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.systolic = datetimeDisplay.trim().toLowerCase(); }
+  }
 
-    const noDataDiastolic = (diastolic && diastolic.resultNoData === true);
-    if (diastolic && !noDataDiastolic) {
-      const {
-        result,
-        status,
-        conceptDisplay,
-        datetimeDisplay,
-      } = diastolic;
+  const noDataDiastolic = (diastolic && diastolic.resultNoData === true);
+  if (diastolic && !noDataDiastolic) {
+    const {
+      result,
+      status,
+      conceptDisplay,
+      datetimeDisplay,
+    } = diastolic;
 
-      if (!isEmpty(result.unit)) { compareUnits.diastolic = result.unit.trim().toLowerCase(); }
-      if (!isEmpty(status)) { compareStatusIsInError.diastolic = checkIsStatusInError(status); }
-      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.diastolic = conceptDisplay.trim().toLowerCase(); }
-      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.diastolic = datetimeDisplay.trim().toLowerCase(); }
-    }
+    if (!isEmpty(result.unit)) { compareUnits.diastolic = result.unit.trim().toLowerCase(); }
+    if (!isEmpty(status)) { compareStatusIsInError.diastolic = checkIsStatusInError(status); }
+    if (!isEmpty(conceptDisplay)) { compareConceptDisplays.diastolic = conceptDisplay.trim().toLowerCase(); }
+    if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.diastolic = datetimeDisplay.trim().toLowerCase(); }
+  }
```

Comment:
this is 1:1 with the `if (systolic && !noDataSystolic)`. Can we bump this into some some sort of utils file so we can write jest tests on this logic & make it reusable?

Reply:
First draft at changing the functions, https://github.com/cerner/terra-clinical/pull/648/commits/5611e410a5b4c6389abf03f7d50fbb975c73143e#diff-72ca9008b342e1caa5acc1466dfa7f20R33
Still need to add some tests but didn't want to go down that rabbit hole until i got feedback.

Reply:
Tests added https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-8b312db510259892da0886d2f909d49eR89